eslint-rules/src/rules/use-jest-extended-matchers-in-tests.ts

```diff
@@ -0,0 +1,85 @@
+import { AST_NODE_TYPES, TSESLint, TSESTree } from '@typescript-eslint/experimental-utils';
+
+import { getClosestAncestorByKind } from '../helpers';
+
+const REPLACEMENTS = [
```

Comment:
Why is this configuration directly included in the lint rule instead as a configuration in the eslintrc.json?